filterExprParser.go

```diff
@@ -868,16 +869,11 @@ func (f *FECompareOp) OutputExpression(lhs Expression, rhs Expression) (Expressi
 }
 
 type FECheckOp struct {
-	Exists  *bool `@"EXISTS" | ( "IS"`
-	Not     *bool `[ @"NOT" ]`
+	Not     *bool `( "IS" [ @"NOT" ]`
```

Comment:
should IS and NOT be two separate fields?

Reply:
The "IS" there is not really a capture, but rather it is needed as part of the syntax detection. So Instead of having a *bool with "IS", it just belongs to the first field of this struct.
Since it's not captured, there's no field variable to put it and once Exists is gone, it now sits with Not.